app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -688,7 +688,15 @@ private void loadData() {
 
     public void onEvent(QueueEvent event) {
         Log.d(TAG, "onEvent(" + event + ")");
-        loadData();
+        switch(event.action) {
+            case ADDED:
+            case ADDED_ITEMS:
+            case SET_QUEUE:
+            case REMOVED:
+            case IRREVERSIBLE_REMOVED:
+            case CLEARED:
+                loadData();
```

Comment:
it makes me nervous to not list all the options here and not say why we don't load data for some.

Can you add the other cases and a comment on why we don't need to load data for them?